### ISSUE TITLE
?!cmd is broken

### ISSUE THREAD

**Issue by DarkPhoenix10** [africateam/tbpp#2]
```
Cannot read property 'nick' of undefined
    at Object.handleMessage (/home/dark/tbpp/handlers/message.js:44:57)
    at Socket.conn.on (/home/dark/tbpp/socket.js:74:12)
    at Socket.emit (events.js:193:13)
    at /home/dark/tbpp/node_modules/socket.io/lib/socket.js:528:12
    at args.(anonymous function) (/usr/lib/node_modules/pm2/node_modules/event-loop-inspector/index.js:138:29)
    at processTicksAndRejections (internal/process/task_queues.js:79:9)
```

**Comment by DarkPhoenix10** [comment]
Yes

**Comment by Wasserboy08** [comment]
did you fix it yet

**Comment by nekofurball** [comment]
I wonder if it's detecting SYSTEM42 as a real user and trying to execute code on a non-real browser.

I'll attempt to try and fix it and document any bugs I find along the way.

Also modifying package.json to update lodash to prevent a vulnerability.

**Comment by nekofurball** [comment]
Patched that bug, but found bug when connecting to the server using HTTP. Wrong variable name. Fixing now.

**Comment by nekofurball** [comment]
Patched both bugs. Pushing both fixes now..

**Comment by nekofurball** [comment]
Pushed fixes in commits c767913 and 0de3543.

Closed issue.
